src/netflow/templates.rs

```diff
@@ -104,7 +104,7 @@ impl TemplatesBox {
         self.templates.remove(key).is_some()
     }
 
-    pub(super) fn add(&mut self, pkt_cnt: u64, src_addr: IpAddr, templates: &[Template]) {
+    pub(crate) fn add(&mut self, pkt_cnt: u64, src_addr: IpAddr, templates: &[Template]) {
```

Comment:
Since this method is only shared within the netflow module, pub(super) seems more appropriate.